CHANGES.rst

```diff
@@ -5,7 +5,7 @@ Changelog
 1.16 (unreleased)
 -----------------
 
-- Nothing changed yet.
+- Fixed data extraction to concatenate all `Pdata` chunks. [@jnptk]
```

Comment:
To be more precise, the fix in the Serializer is only for ATFileFields from the Archetypes contenttype system, where the binary Data of the FileField is stored in the ZODB instead of Blobstorage (using plone.app.blob).  Most Plone sites with Archetypes would/should have been migrated or on install already use Blob based filefield storage in Plone 4. 

A mnore precies changelog line could be along the lines of:

- Fixed data extraction of Archetypes ATFilefields to concatenate all 'Pdata' chunks, when the FileField data was still stored in Filestorage and not not migrated to use blobstorage (using plone.app.blobs).